src/util/desktop-notifier.js

```diff
@@ -1,22 +1,35 @@
 /* @flow */
-import {NotificationCenter} from 'node-notifier';
+import defaultNotifier from 'node-notifier';
 
-const defaultNotifier = new NotificationCenter({
-  withFallback: true,
-});
+import {createLogger} from './logger';
+
+const log = createLogger(__filename);
 
 type desktopNotificationsParams = {
   title: string,
   message: string,
   icon?: string,
-  notifier?: typeof NotificationCenter,
+};
+
+type desktopNotificationsOpts = {
+  notifier?: typeof defaultNotifier,
+  logger?: typeof log,
 };
 
 export function desktopNotifications(
   {
-    title, message, icon, notifier = defaultNotifier,
-  }: desktopNotificationsParams
+    title, message, icon,
+  }: desktopNotificationsParams,
+  {
+    notifier = defaultNotifier,
+    logger = log,
+  }: desktopNotificationsOpts = {}
 ): void {
 
-  notifier.notify({title, message, icon});
+  notifier.notify({title, message, icon}, (err, res) => {
+    if (err) {
+      logger.debug(`notifier error: ${err.message},` +
```

Comment:
It would be probably better to capitalize the first letter of this comment (e.g. `Desktop notifier error: ...`)

Also, I'm not sure if the log level is the right one, probably @kumar303 was suggesting to log the error even when web-ext is not running in verbose mode.

 @kumar303 am I right?